### PR TITLE
add partial load telemetry for experiement.

### PR DESCRIPTION
**Pull request by heejaechang**
we might delete this telemetry once experiement is done and a telemetry is added to the platform.

for now, we will have one in our side which tell us how long it took and how many times we are in the partial load mode.

### CODE REVIEW

**Comment by heejaechang** [comment]
tagging @dibarbet

**Comment by heejaechang** [comment]
tagging @jinujoseph - adding telemetry to track partial load numbers among feature group.

File: src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioWorkspaceStatusServiceFactory.cs [hunk 1]
```diff
@@ -77,17 +78,23 @@ public Service(IAsyncServiceProvider2 serviceProvider, IAsynchronousOperationLis
...
-                //       for now, use WithCancellation to indirectly add cancellation
-                await status.WaitForCompletionAsync().WithCancellation(cancellationToken).ConfigureAwait(false);
+                    var status = await GetProgressStageStatusAsync(cancellationToken).ConfigureAwait(false);
+                    if (status == null)
+                    {
+                        return;
+                    }
+
+                    var completionTask = status.WaitForCompletionAsync();
+                    Logger.Log(FunctionId.PartialLoad_FullyLoaded, KeyValueLogMessage.Create(LogType.Trace, m => m["InitiallyFullyLoaded"] = completionTask.IsCompleted));
```
Comment (dibarbet) [inline src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioWorkspaceStatusServiceFactory.cs]: Minor - would maybe rename to 'AlreadyFullyLoaded' ?

Comment (heejaechang) [reply src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioWorkspaceStatusServiceFactory.cs]: sure.

**Pull request merged**